### PR TITLE
Add KoP grafana dashboard

### LINKED ISSUE #1102: [Feature] Add grafana dashboard for KoP

### PR DESCRIPTION
**Pull request by Demogorgon314**
Fixes #1102

### Motivation

Add KoP Grafana dashboard, so users can easily import the dashboard to Grafana.

### Documentation

Need to update docs? 

- [x] `doc`

### CODE REVIEW

File: grafana/README.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+* KoP >= 2.9.0
+* Grafana >= v8.4.3
+* Prometheus >= v2.0.0
+
+A Prometheus data source needs to be [added](https://prometheus.io/docs/visualization/grafana/#using) before installing the dashboard.
+
+## Installing the Dashboard
+
+In the Grafana UI complete the following steps:
```
Comment (Huanli-Meng) [inline grafana/README.md:17]:
```suggestion
To install the Grafana Dashboard, follow these steps in the Grafana UI:
```
One question: which word is more accurate, GUI, console, or UI?

File: grafana/README.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+1. Use the *New Dashboard* button and click *Import*.
```
Comment (Huanli-Meng) [inline grafana/README.md:19]:
```suggestion
1. Click **New Dashboard** and then click **Import**.
```

File: grafana/README.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+2. Upload `dashboard.json` or copy and paste the contents of the file in the textbox and click *Load*.
```
Comment (Huanli-Meng) [inline grafana/README.md:20]:
```suggestion
2. Upload the `dashboard.json` JSON file or copy and paste the contents of the file in the textbox and then click **Load**.
```
One question, where can we add a link to the `dashboard.json` JSON file?

File: grafana/README.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+3. Set the Prometheus data source and click *Import*.
```
Comment (Huanli-Meng) [inline grafana/README.md:21]:
```suggestion
3. Set the Prometheus data source and click **Import**.
```

**Review by Huanli-Meng — APPROVED** [review]
Leave some comments. PTAL

File: grafana/README.md (hunk 1 above)
Comment (Demogorgon314) [reply grafana/README.md:17]: Grafana provided a Web interface. Should we call it UI?

**Review by Huanli-Meng — APPROVED** [review]
LGTM

**Review by BewareMyPower — APPROVED** [review]

**Pull request merged**